README.md

```diff
@@ -151,6 +156,14 @@ At runtime, attempting to create a `Record` with a key that is not a `string` or
 
 At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `symbol`, `boolean`, `bigint`, `undefined`, or `null`.
 
+However whenever the engine can figure out at parse time that something is wrong, it should fail at that moment:
```

Comment:
are we sure about this? This runs counter to my [babel implementation](https://github.com/babel/babel/pull/10865), where any expressions inside a record are valid at parse time (including method definitions), but fail at runtime.

Reply:
Discussions in #88 seems to indicate that we would prefer parse time errors if possible. The Babel implementation does not need to be fully compliant with that or could eventually be improved later.

Reply:
I'm confused. I think the conclusion on #88 was runtime errors.

Reply:
I reread it and yes it was indeed the contrary